src/main/java/duke/ui/Ui.java

```diff
@@ -0,0 +1,24 @@
+package duke.ui;
+
+/** User Interface Manager. */
+public class Ui {
+
+    public static void printIntro() {
+        String intro = "Hello! I'm Duke" + System.lineSeparator() + "What can I do for you?";
+        System.out.println(wrap(intro));
+    }
+
+    public static void printMessage(String message) {
+        System.out.println(wrap(message));
+    }
+
+    public static String wrap(String string) {
+        return "____________________________________________________________"
+                + System.lineSeparator()
+                + string
+                + System.lineSeparator()
+                + "____________________________________________________________"
+                + System.lineSeparator();
```

Comment:
Neat placement of the components of the return String. For example, you placed `+ string` on its own line even though it was a short statement, but this allows readers to see each element individually more clearly